src/plugins/minecraft.rs

```diff
@@ -0,0 +1,37 @@
+use async_minecraft_ping::Server;
+
+use crate::prelude::*;
+
+pub struct MinecraftPlugin {}
```

Comment:
I think you can just do
```suggestion
pub struct MinecraftPlugin;
```